docs/components/integration-framework/connectors/custom-built-connectors/connector-sdk.md

```diff
@@ -4,17 +4,677 @@ title: Connector SDK
+[custom connector template repository on GitHub](https://github.com/camunda/connector-template).
+This template is a [Maven](https://maven.apache.org/)-based Java project, and you can use it the
+way that fits best into your development flow:
+
+- _Create your own GitHub repository_: Click the **Use this template** button and follow the steps.
+  You can manage code changes in your new repository afterward.
+- _Experiment locally_: Check out the source code to your local machine using [Git](https://git-scm.com/).
+  You won't be able to check in code changes to the repository due to restricted write access.
+- _Fetch the source_: Download the source code as a ZIP archive using **Code** > **Download ZIP**.
+  You can adjust and manage the code the way you like afterward, using your chosen source code
+  management tools.
+
+If you want to manualy set up your connector project, include the following dependency to use the SDK.
+Make sure to adhere to the project outline detailed in the next section.
+
+<Tabs groupId="dependency" defaultValue="maven" values={
+[
+{label: 'Maven dependency', value: 'maven' },
+{label: 'Gradle dependency', value: 'gradle' }
+]
+}>
+
+<TabItem value='maven'>
+
+```xml
+<dependency>
+  <groupId>io.camunda.connector</groupId>
+  <artifactId>connector-core</artifactId>
+  <version>0.1.0</version>
+</dependency>
+```
+
+</TabItem>
+
+<TabItem value='gradle'>
+
+```yml
+implementation 'io.camunda.connector:connector-core:0.1.0'
+```
+
+</TabItem>
+</Tabs>
+
+### Project outline
+
+A connector consists of multiple parts that create a reusable building block for modeling and
+runtime behavior. In essence, a connector includes the following things:
+
+```
+my-connector
+├── element-templates/
+│   └── template-connector.json        (1)
+├── src/main
+│   ├── java/io/camunda/connector      (2)
+│   │   ├── MyConnectorFunction.java   (3)
+│   │   ├── MyConnectorRequest.java    (4)
+│   │   └── MyConnectorResult.java     (5)
+│   └── resources/META-INF/services
+│       └── io.camunda.connector.api.ConnectorFunction (6)
+└── pom.xml (7)
+```
+
+Regarding modeling building blocks, the connector provides at least one
+[connector template](./connector-templates.md) with **(1)**.
+
+You provide the runtime logic as Java source code under a directory like **(2)**.
+Usually, a connector runtime logic consists of
+
+- Exactly one implementation of a `ConnectorFunction` with **(3)**
+- At least one input data object like **(4)**
+- At least one result object like **(5)**
+
+To make your connector function detectable, you must expose your function class name in the
+[`ConnectorFunction` SPI implementation](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/ServiceLoader.html)
+with **(6)**.
+
+A configuration file like **(7)** manages the project setup, including dependencies.
+In this example, we include a Maven project's `POM` file. Using other build tools like
+[Gradle](https://gradle.org/) is possible as well.
```

Comment:
Minor tweak:
Update text from "Using other build tools like [Gradle](https://gradle.org/) is possible as well."
to "Other build tools like [Gradle](https://gradle.org/) can also be used."